bind_uploader.go

```diff
@@ -41,7 +38,7 @@ func (bu *bindUploader) upload(bindings []driver.NamedValue) (*execResponse, err
 	bu.fileCount = 0
 	var data *execResponse
 	for rowNum < len(bindingRows) {
-		for numBytes < inputStreamBufferSize && rowNum < len(bindingRows) {
+		for rowNum < len(bindingRows) && numBytes + len(bindingRows[rowNum]) < inputStreamBufferSize {
```

Comment:
note that the conditionals' ordering had to be swapped, since otherwise you can get a slice index out of bounds on the "bindingRows[rowNum]" part